src/com/ppm/integration/agilesdk/connector/octane/OctaneRequestIntegration.java

```diff
@@ -21,6 +23,7 @@
 import com.ppm.integration.agilesdk.model.AgileEntity;
 import com.ppm.integration.agilesdk.model.AgileEntityField;
 import com.ppm.integration.agilesdk.model.AgileEntityFieldInfo;
+import com.ppm.integration.agilesdk.model.AgileEntityFieldInfo.AgileEntityFieldType;
 import com.ppm.integration.agilesdk.model.AgileEntityFieldValue;
 
 import edu.emory.mathcs.backport.java.util.Collections;
```

Comment:
You shoudln't use this Collections class (I know, it's not your code but this needs to be fixed)